### PR TITLE
First mobile project skeleton

### PR DESCRIPTION
**Pull request by GrebilCruz**
First skeleton project, headline, and navigation bar hover and social media added with the greeting backgrounds and style with links to the icons 
Fixed all the linterns and now they'r working properly, fixed the error witn the YML, thanks alot the x.17 worked that was the solution appreciate it.
added hover an responsive icons
NOTE: **_I need help to solve the lintern error, I've tried a lot of things and I don't know what is going on_**

### CODE REVIEW

**Review by keroloussamy — COMMENTED** [review]
Hi @GrebilCruz, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linters error, and I think that because you didn't add these two file [.hintrc](https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc) and [.stylelintrc.json](https://github.com/microverseinc/linters-config/blob/master/html-css/.stylelintrc.json) it's mandatory to add them.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by keroloussamy** [comment]
Hello @GrebilCruz  try to change node-version to `node-version: "17.x"` in your linters.yml file line 28, it will solve the issue. And Contact me if you encounter any problems

File: home.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  </header>
+  <section id="headline">
+    <div class="greeting">
+      <h1>I'm Grebil, Glad to See you!</h1>
+    </div>
+    <p>I'm a software developer! I can help you build a product, feature or website Look through some of my work and
+      experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <p class="connect">
+      Let's Connect!
+    </p>
```
Comment (CollinsTatang) [inline home.html]: - [ ] Please remove the `p tag` here and use any of the `h2, h3, h4, h5, h6 tag`.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  background-repeat: no-repeat;
+  padding-top: 206px;
+  padding-left: 26px;
+  padding-right: 26px;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+}
```
Comment (CollinsTatang) [inline style.css]: - [ ] Please reduce the `padding-top: 206px` it actually generating a lot of space from the `header` to the headline `section`.
### **Take a look**
padding

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @GrebilCruz, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @GrebilCruz   , 

Your project is complete!  You have implemented successfully all the suggestions asked by the previous reviewer. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well implemented Figma design.

### Optional suggestions :arrow_down_small: 

- [ ] The commit message should be written in the imperative tense. `Ex: "Fix bug" instead of "Fixed bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

- [ ] For readability, please add two spaces of indentation. Do not use the tab key. You did it throughout your code in the `HTML` file. Please fix it. [For the HTML & CSS best practices, ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)we should follow the rules.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**